CMakeLists.txt

```diff
@@ -1075,6 +1078,8 @@ if(APPLE)
 endif()
 
 # Add the install target
+install(TARGETS libcsound_interface EXPORT CsoundExports)
```

Comment:
So this export is just the libcsound public headers right?

Reply:
Yes